sdk/textanalytics/Azure.AI.TextAnalytics/README.md

```diff
@@ -19,6 +19,16 @@ Install the Azure Text Analytics client library for .NET with [NuGet][nuget]:
 ```PowerShell
 dotnet add package Azure.AI.TextAnalytics
 ```
+> Note: This version of the client library defaults to the `v3.1-preview.5` version of the service.
+
+This table shows the relationship between SDK versions and supported API versions of the service:
+
+|SDK version|Supported API version of service
+|-|-
+|1.0.0 | 3.0
+|1.0.1 | 3.0
+|5.0.0 (latest GA) | 3.0
+|5.1.0-beta.7 (latest Beta) | 3.0, 3.1-preview.5
```

Comment:
maybe just me, but I would order this latest -> oldest. Also maybe you can do 1.0.X to condense rows?

Reply:
makes sense. I will reorder. 
As for the specific versions, I think for now I will leave it explicit, but maybe join into a single row.